cypress/integration/vizsurvey.spec.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+
+describe('vizsurvey', () => {
```

Comment:
@yahelnachum  can you add instructions on how to run the cypress tests?  I would like to try and run those locally.  Thanks.